src/halmos/cheatcodes.py

```diff
@@ -289,12 +289,42 @@ def snapshot_state(
     return ByteVec(balance_hash + code_hash + storage_hash + path_hash)
 
 
-def create_calldata_contract(ex, arg, sevm, stack):
+def _get_contract_name(ex, arg) -> tuple[str | None, str | None]:
+    # TODO: support symbolic target using sevm.resolve_address_alias()
+    addr = con(
+        int_of(extract_bytes(arg, 4, 32), "symbolic address for SVM.createCalldata()"),
+        size_bits=160,
+    )
```

Comment:
```suggestion
    addr = con_addr(
        int_of(arg.get_word(4), "symbolic address for SVM.createCalldata()"),
    )
```

Reply:
nice, let me open a separate pr that fixes this throughout the codebase.

Reply:
fixed: https://github.com/a16z/halmos/pull/519